### PR TITLE
Initialize UptimeCalculator in TestPeer

### PR DESCRIPTION
**Pull request by joshua-kim**
## Why this should be merged

TestPeer breaks if it handles an inbound ping message because it has a `nil` UptimeCalculator

## How this works

Initializes UptimeCalcualtor

## How this was tested

CI

### CODE REVIEW

**Comment by dhrubabasu** [comment]
Can we add a test that fails on dev but passes with this change?

File: network/peer/test_peer.go [hunk 1]
```diff
@@ -117,6 +118,7 @@ func StartTestPeer(
 			MaxClockDifference:   time.Minute,
 			ResourceTracker:      resourceTracker,
 			IPSigner:             NewIPSigner(signerIP, tls),
+			UptimeCalculator:     uptime.NoOpCalculator,
```
Comment (StephenButtolph) [inline network/peer/test_peer.go:121]: Super small nit, but can we put the `UptimeCalculator` between `ResourceTracker` and `IPSigner`? That's how it's defined in the `Config`, so I think it's easier to make sure we're populating the expected fields if we keep them in the same order.

**Comment by StephenButtolph** [comment]
Not sure how easy this would be to make a unit test for... But we could probably make a test for it in the e2e tests (as we already have a full node spun up there to connect to).

**Comment by joshua-kim** [comment]
> Not sure how easy this would be to make a unit test for... But we could probably make a test for it in the e2e tests (as we already have a full node spun up there to connect to).

Unit might be a bit annoying since it would increase build time but I think something in e2e sounds reasonable

**Review by StephenButtolph — APPROVED** [review]

**Review by dhrubabasu — APPROVED** [review]

**Review by patrick-ogrady — APPROVED** [review]

**Pull request merged**